apps/projects/app/components/Content/Settings.js

```diff
@@ -36,13 +36,13 @@ const GitHubConnect = ({ onLogin, onLogout, status }) => {
   const bodyText = auth ? (
     <Text size="large" css="display: flex; align-items: center">
       Logged in as <img src={user.avatarUrl} alt="user avatar" css="margin: 8px; width: 50px; border-radius: 50%;" />
-      <SafeLink
+      <Link
         href={user.url}
         target="_blank"
-        style={{ textDecoration: 'none', color: `${theme.accent}` }}
+        style={{ textDecoration: 'none', color: `${theme.link}` }}
       >
         {user.login}
-      </SafeLink>
+      </Link>
```

Comment:
Why get rid of `SafeLink`?

Reply:
It complains about being deprecated (and replaced by 'Link').